README.md

```diff
@@ -52,5 +52,20 @@ Or, individually configure the rules you want to use under the rules section.
 -   [`package-json/order-properties`](docs/rules/order-properties.md): Require top-level properties to be in a conventional order (`"name"`first, etc.).
 -   [`package-json/sort-collections`](docs/rules/sort-collections.md): Keep sub-collections like `"dependencies"` and `"scripts"` in alphabetical order.
 -   [`package-json/valid-package-def`](docs/rules/valid-package-def): Validate `package.json` files against the NPM specification.
+-   [`package-json/valid-package-def`](docs/rules/valid-package-def): Validates the casing for `file:` and `link:` dependencies in the `package.json` files. If the casing of these paths are incorrect yarn does not sub out the link when releasing. An example:
+
+    This:
+    ```
+    "devDependencies": {
+            "some-package": "link:../folder",
+    }
+    ```
+    Should be:
+    ```
+    "devDependencies": {
+            "some-package": "link:../Folder",
+    }
+    ```
```

Comment:
[Docs]

* It's much more common for npm projects to have lowercased package names. What do you think about switching the order here so that `link:../folder` is mentioned as preferred?
* This explanation confused me at first because it never explains _why_ one should be preferred. Can you add some mention of what the folder case is correctly?
    * If there's a good docs link, that might be nice too?